scripts/add_export_paths_to_product_repos.py

```diff
@@ -0,0 +1,48 @@
+import os
+import re
+import sys
+
+sys.path.append(os.path.dirname(os.path.dirname(__file__)))
+
+from fastapi_sqla import open_session
+from sqlalchemy import select
+from sqlalchemy.orm import joinedload
+
+from alws.models import Product
+from alws.utils.fastapi_sqla_setup import sync_setup
+
+
+def main():
+    sync_setup()
+    with open_session() as session:
+        for product in (
+            session.execute(
+                select(Product)
+                .where(Product.is_community.is_(True))
+                .options(joinedload(Product.repositories), joinedload(Product.platforms))
+            )
+            .scalars()
+            .unique()
+            .all()
+        ):
+            if not product.repositories:
+                continue
+            platform_names = '|'.join((platform.name for platform in product.platforms))
```

Comment:
Double brackets around the generator are redundant. You can use single brackets.
For example:

```
>>> ', '.join(str(i) for i in range(10))
'0, 1, 2, 3, 4, 5, 6, 7, 8, 9'
>>>
```

Reply:
Fixed